### ISSUE TITLE
/comments redirect to /login

### ISSUE THREAD

**Issue by Jipem** [laravelista/comments#64]
Hello,

I just added laravelista/comments 3.1.0 to my project based on Laravel 6. But after a long moment of research, whatever the action of comment (add, delete, update), the route /comments redirects to /login .

I did not modify the config file (and also tested with "guest_commenting" to true and false).
My config Laravel is "basic".
The migration has been launched.
The "auth" middleware is working properly on the other controller / route.

In my App\Models\User I added:
```
use Laravelista\Comments\Commenter;
class User extends Authenticatable
{
    use Notifiable, Commenter;
}
```

In my App\Models\Profile I added:
```
use Laravelista\Comments\Commentable;
class Profil extends Model
{
    use Commentable;
}
```

In my Controller I have:
```
$comment = Profile::find($id);
```

In my view file.blade.php I added:
```
@comments ([
     'model' => $ comment
])
```

An idea ? 
Thank you in advance !

**Comment by mabasic** [comment]
I'm going to look into it. Give me a few minutes to setup a L6 project

**Comment by mabasic** [comment]
There is something funny going on here. 

`auth()->user()` returns `null` after I log in. That is why we are getting redirected to `/login` and then back to where ever you were.

**Comment by mabasic** [comment]
Fixed in `3.1.1`.

**Comment by Jipem** [comment]
It works perfectly (with 3.1.1), thank you @mabasic  :)
